backstop.json

```diff
@@ -37,25 +37,19 @@
       "removeSelectors": ["#frame"]
```

Comment:
small copy update above on line 28, can we update to “Installation” instead of install with npm to match the site